gradio/interface.py

```diff
@@ -735,18 +736,35 @@ class TabbedInterface(Blocks):
     """
 
     def __init__(
-        self, interface_list: List[Interface], tab_names: Optional[List[str]] = None
+        self,
+        interface_list: List[Interface],
+        tab_names: Optional[List[str]] = None,
+        theme: str = "default",
+        analytics_enabled: Optional[bool] = None,
+        title: str = "Gradio",
+        css: Optional[str] = None,
     ):
         """
         Parameters:
             interface_list: a list of interfaces to be rendered in tabs.
             tab_names: a list of tab names. If None, the tab names will be "Tab 1", "Tab 2", etc.
+            theme: which theme to use - right now, only "default" is supported.
+            analytics_enabled: whether to allow basic telemetry. If None, will use GRADIO_ANALYTICS_ENABLED environment variable or default to True.
+            mode: a human-friendly name for the kind of Blocks interface being created.
+            title: The tab title to display when this is opened in a browser window.
+            css: custom css or path to custom css file to apply to entire Blocks
         Returns:
             a Gradio Tabbed Interface for the given interfaces
         """
+        super().__init__(
+            theme=theme,
+            analytics_enabled=analytics_enabled,
+            mode="tabbed_interface",
+            title=title,
```

Comment:
The UI won't display the title so maybe we shouldn't add it to the init until it can? I feel like it will confuse users.

```python
import gradio as gr

def test(text1):
    return [(text1, "good"), ("\n", None), (text1, "good")]

demo = gr.Interface(
    test,
    "text",
    gr.HighlightedText(elem_id="htext"),
    title="Tab 1",
)

def test(text1):
    return [(text1, "good"), ("\n", None), (text1, "good")]

demo2 = gr.Interface(
    test,
    "text",
    gr.HighlightedText(elem_id="htext2"),
    title="Tab 2",
)

gr.TabbedInterface([demo, demo2], css="#htext span {white-space: pre} #htext2 span {white-space: pre}",
                   title="Tabbed Interface").launch()
```

![image](https://user-images.githubusercontent.com/41651716/182981551-a6cfb8bf-5649-49a2-911a-f8d758784040.png)


The title is set properly in the config though so I think it's just a front-end issue

Reply:
Ah so the title here just sets the Tab title. (It's more similar to a Blocks title as opposed to the Interface title). But I agree it might lead to confusion so will remove.